waspc/data/Generator/templates/react-app/src/test/vitest/helpers.tsx

```diff
@@ -9,27 +10,39 @@ import { Query } from '../../queries'
 import config from '../../config'
 import { HttpMethod } from '../../types'
 
+export type Route = { method: HttpMethod; path: string }
+
+export type MockQuery = <Input, Output, MockOutput extends Output>(
+  query: Query<Input, Output>,
+  resJson: MockOutput
+) => void
+
+export type MockApi = (route: Route, resJson: unknown) => void
```

Comment:
The rule of thumb for exporting types is: _If users can dig it out themselves (e.g., using utility types like `Parameters`), make it public._